roles/download/defaults/main.yml

```diff
@@ -91,6 +91,16 @@ fluentd_image_tag: "{{ fluentd_version }}"
 kibana_version: "v4.6.1"
 kibana_image_repo: "gcr.io/google_containers/kibana"
 kibana_image_tag: "{{ kibana_version }}"
+grafana_version: "v4.0.2"
```

Comment:
would you mind prefixing all of these with `heapster_` to help with namespacing?